docs-ui/components/shareIssue.stories.js

```diff
@@ -0,0 +1,52 @@
+// eslint-disable-next-line sentry/no-react-hooks
+import React, {useState} from 'react';
```

Comment:
can we already use hooks? 😃👏

Reply:
We have been avoiding them in the application code so far. I don't think we made decisions about what to do in non-application code.

Reply:
This was easier to write as a hook inside storybook for the example

Reply:
Removed the hooks